packages/apollo/package.json

```diff
@@ -15,9 +15,9 @@
     "url": "git+https://github.com/nestjs/graphql.git"
   },
   "scripts": {
-    "test:e2e": "jest --config ./tests/jest-e2e.ts --runInBand && yarn test:e2e:fed2",
-    "test:e2e:fed2": "jest --config ./tests/jest-e2e-fed2.ts --runInBand",
-    "test:e2e:dev": "jest --config ./tests/jest-e2e.ts --runInBand --watch"
+    "test:e2e": "node --expose-gc ../../node_modules/.bin/jest --config ./tests/jest-e2e.ts --runInBand && yarn test:e2e:fed2",
+    "test:e2e:fed2": "node --expose-gc ../../node_modules/.bin/jest --config ./tests/jest-e2e-fed2.ts --runInBand",
+    "test:e2e:dev": "node --expose-gc ../../node_modules/.bin/jest --config ./tests/jest-e2e.ts --runInBand --watch"
```

Comment:
Appreciate you wanting to show that there's not a memory leak, but is  it really necessary to change the commands themselves rather than possibly updating the CI to watch for this?

Reply:
I found a solution to use `global.gc()` without node flags. Check out the new PR.